packages/graph/hash_graph/.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgres://postgres:postgres@localhost/postgres
```

Comment:
It would be cool if we could somehow leverage the existing `.env`. May also lead to some indirection that we'd need to document. 

Currently, the migrations use the `HASH_MIGRATION_PG_URL` env var, which means we are defining the connection string multiple places.

Changing this would mean we can use `dotenv` in the binary to fetch the root level config

Reply:
Yeah I've added this as a short-circuit for immediate development. There might be a way to override the environment variable for example but the macro's error message at least was explicit about wanting `DATABASE_URL`. I also am not actually convinced that we want `.env` files shared across the packages, because it doesn't make sense to have it for the hEngine for example, so I'd suggest we leave it as is, and delete the file when we stop using the `query!` macro, and then figure out a proper approach when we need env-vars again